inc/deployfile.class.php

```diff
@@ -435,16 +435,11 @@ static function displayAjaxValues($config, $request_data, $rand, $mode) {
       Html::showCheckbox(array('name' => 'p2p', 'checked' => $p2p));
       echo "</td>";
       echo "</tr><tr>";
-      echo "<th>".__("retention days", 'fusioninventory')."</th>";
+      echo "<th>".__("retention", 'fusioninventory').
+                  " - ".__("Minute(s)", 'fusioninventory')."</th>";
       echo "<td>";
-      /*
-       * TODO: use task periodicity input to propose days, months and years
-       */
-      Dropdown::showNumber('p2p-retention-duration', array(
-             'value' => $p2p_retention_duration,
-             'min'   => 0,
-             'max'   => 400)
-      );
+      
+      echo "<input type='number' name='p2p-retention-duration' value='$p2p_retention_duration' />";
```

Comment:
It's better to keep Dropdown::showNumber to disable possibility to user to put other value than integer value